includes/data-stores/class-wc-admin-notes-data-store.php

```diff
@@ -33,7 +33,7 @@ public function create( &$note ) {
 			'source'  => $note->get_source(),
 		);
 
-		$encoding_options = defined( 'JSON_FORCE_OBJECT' ) ? JSON_FORCE_OBJECT : 0;
+		$encoding_options = defined( 'JSON_FORCE_OBJECT' ) ? JSON_FORCE_OBJECT : 0; // phpcs:ignore PHPCompatibility.PHP.NewConstants
```

Comment:
Can this be refactored to not use a linter ignore?  This is something for another PR.  Does the rule make sense?  I've found that when you start making exceptions it becomes a slippery slope.

Reply:
Overkill. Removed in https://github.com/woocommerce/wc-admin/pull/507/commits/931b45f71d97a3ea5ee15244c68355514e947d95